__tests__/__snapshots__/plugin.test.js.snap

```diff
@@ -243,7 +243,7 @@ function Box() {
     };
 
     _f._closure = {
-      offset: offset
+      offset
```

Comment:
Are we sure this change doesn't break anything? I'm trying to understand why sometimes there's `x: x` in `_f._closure` but sometimes just `offset` and not `offset: offset`.